README-dev.md

```diff
@@ -50,13 +50,17 @@ IDE [here](https://black.readthedocs.io/en/stable/integrations/editors.html).
 
 ## ..create a new release
 
-Using the [Github cli](https://cli.github.com/), run:
-
-```shell
-gh release create 0.0.2 --generate-notes
-```
-
-which will trigger a build and release to https://pypi.org/project/mpyl/
+§§1. Check the latest [release number](https://github.com/Vandebron/mpyl/releases) 
+2. Add the new release number to [mpyl/cli/releases/releases.txt](src/mpyl/cli/releases/releases.txt)
+3. Create a new release notes file in [releases/notes/](releases/notes/) and name it `<version>.md`
+Noteworthy changes should be added to this file. Think: new cli commands, new features, breaking changes, upgrade 
+instructions, etc.
+4. Merge all PRs that you want to include in the release to `main`
```

Comment:
nit: this should be step 1 I would say

Reply:
I'm suggesting to consider creating release notes for each PR. Hope this makes that a bit clearer: https://github.com/Vandebron/mpyl/pull/194/commits/63a2c7456b88f302340001961bd3f18a32c73b3f

Reply:
Ahh okay!